src/core.ts

```diff
@@ -1,4 +1,4 @@
-import expect from 'jest-matchers';
+import expect from 'expect';
 import jestMock from 'jest-mock';
 import {addEventHandler} from 'jest-circus/build/state';
```

Comment:
we should make sure to export what you need from the top level as well

Reply:
Like export all named exports from `expect`? `export * from 'expect';`

Reply:
No, I meant that you shouldn't have to reach into `build/`

Reply:
Ohh haha gotcha! Totally agree! And while we're on that topic there should probably be type generation for `jest-circus` as well. Its written in TypeScript but due to the lack of type definitions you need to define them yourself: https://github.com/kvendrik/jest-lite/blob/master/config/jest-circus.d.ts.

What do you think? Shall I open issues for this so we can look into that?

Reply:
`jest-circus@24` comes with bundled ts types :)

Reply:
Opened a PR for the top level exports [here](https://github.com/facebook/jest/pull/8748).